File: 5 - Funções/PHP/funcoes05.php
<?php

#Escreva uma função chamada soma_array que retorna a soma dos elementos de um array.

function soma_array($array) {
  $somaDosElementos = 0;
  foreach ($array as $indice => $elemento) {
    $somaDosElementos += $elemento;
  }
  return $somaDosElementos;
}

File: 5 - Funções/Ruby/funcoes05.rb
#Escreva uma função chamada soma_array que retorna a soma dos elementos de um array.

def soma_array(array)
	somaDosElementos = 0
	for i in 0..array.size - 1 do
		somaDosElementos += array[i]
	end
	return somaDosElementos
end
